docs/sql-reference/functions-reference/semi-structured-functions/array-functions.md

```diff
@@ -29,17 +29,16 @@ ALL_MATCH(<func>, <arr>)
 
 ```sql
 SELECT
-    ALL_MATCH(x -> x > 0, [1,2,3,9]) AS res;
+	ALL_MATCH(x -> x > 0, [ 1, 2, 3, 9 ]) AS res;
```

Comment:
Does this create a problem on cut and paste or some other issue? The style we started aiming for is to space in by 2 spaces. The reason is that we have limited width in the code pane for viewing, so we're trying to conserve space.

Reply:
I was following what the "Format SQL" command does in the UI. Since it uses a tab, maybe you can control the tab width settings for all of the docs? But failing that, two spaces is totally fine too

Reply:
https://bitbucket.org/packdb/packui/src/28ddc0f8d1c0d59a5e43c3fa6ed523ec0d6873d8/src/components/QueryEditor/hooks/useFormatSQL.ts#lines-8 is where the tab is configured. The library's default behavior is two spaces